### ISSUE TITLE
Question on grpc_wrap.cc union comment

### ISSUE THREAD

**Issue by dfreese** [tikv/grpc-rs#519]
With https://github.com/rust-lang/rust/pull/77547 stable as of rust 1.49, I was curious, is [this comment](https://github.com/tikv/grpc-rs/blob/master/grpc-sys/grpc_wrap.cc#L36-L38) still valid?

**Comment by BusyJay** [comment]
No, it's not valid anymore. We can start migrating the c code to rust.
